### DISCUSSION TITLE
What is the operating system you're using to cross-compile CMLFS?

### DISCUSSION THREAD

**Discussion by takusuman** [dslm4515/CMLFS#48] [General]
I'm thinking of trying to compile CMLFS here to actually try to find errors and actually try to help in something on the project, but i'm in doubt about how to build it correctly.
What is the host you're using? I'm currently using Devuan Beowulf (equivalent to Debian 10).

I think things like this are worth to be put in a documentation-- or better, an index containing tested/compatible hosts for cross-compiling it.

**Comment by takusuman** [comment]
PS: I think this would be useful to put in MLFS' documentation too :^)

**Comment by owl4ce** [comment]
Mine is Gentoo :)

**Comment by dslm4515** [comment]
Ever since I got MLFS working years ago, I've always been developing on MLFS. Before that, it was LFS with no package manager.

I no longer have a recent LFS system running (maybe 7.0?).

I did test CMLFS on a Glibc host via Ubuntu...both toolchains build and work.

**Comment by takusuman** [comment]
> Ever since I got MLFS working years ago, I've always been developing on MLFS. Before that, it was LFS with no package manager.
> 
> I no longer have a recent LFS system running (maybe 7.0?).
> 
> I did test CMLFS on a Glibc host via Ubuntu...both toolchains build and work.

So i think it shall work in Devuan 10 too. :^)
I will test this weekend.

**Comment by andzai1995** [comment]
Ubuntu 21.04. Earlier I have built MLFS on Astra Linux 2.12.40. Everything went well.
